From 32380f229d6c8fb6935f6c60db5af4f022df5381 Mon Sep 17 00:00:00 2001
From: Ahmad Samir <a.samirh78@gmail.com>
Date: Tue, 7 Oct 2025 22:08:31 +0300
Subject: [PATCH 350/553] tst_QArrayData: port away from QPair

Pick-to: 6.8 6.5
Task-number: QTBUG-115841
Change-Id: Ia5d90782f12c27715cec3a76c100c3563ac82b0e
Reviewed-by: Thiago Macieira <thiago.macieira@intel.com>
(cherry picked from commit c580fcbdf835b9b54709a1bbc873805b4764425a)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 tests/auto/corelib/tools/qarraydata/tst_qarraydata.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/auto/corelib/tools/qarraydata/tst_qarraydata.cpp b/tests/auto/corelib/tools/qarraydata/tst_qarraydata.cpp
index 5804aeeb1e5..bf0781390eb 100644
--- a/tests/auto/corelib/tools/qarraydata/tst_qarraydata.cpp
+++ b/tests/auto/corelib/tools/qarraydata/tst_qarraydata.cpp
@@ -584,7 +584,7 @@ void tst_QArrayData::typedData()
     {
         Deallocator keeper(sizeof(char),
                 alignof(QTypedArrayData<char>::AlignmentDummy));
-        QPair<QTypedArrayData<char> *, char *> pair = QTypedArrayData<char>::allocate(10);
+        std::pair<QTypedArrayData<char> *, char *> pair = QTypedArrayData<char>::allocate(10);
         QArrayData *array = pair.first;
         keeper.headers.append(array);
 
@@ -604,7 +604,7 @@ void tst_QArrayData::typedData()
     {
         Deallocator keeper(sizeof(short),
                 alignof(QTypedArrayData<short>::AlignmentDummy));
-        QPair<QTypedArrayData<short> *, short *> pair = QTypedArrayData<short>::allocate(10);
+        std::pair<QTypedArrayData<short> *, short *> pair = QTypedArrayData<short>::allocate(10);
         QArrayData *array = pair.first;
         keeper.headers.append(array);
 
@@ -624,7 +624,7 @@ void tst_QArrayData::typedData()
     {
         Deallocator keeper(sizeof(double),
                 alignof(QTypedArrayData<double>::AlignmentDummy));
-        QPair<QTypedArrayData<double> *, double *> pair = QTypedArrayData<double>::allocate(10);
+        std::pair<QTypedArrayData<double> *, double *> pair = QTypedArrayData<double>::allocate(10);
         QArrayData *array = pair.first;
         keeper.headers.append(array);
 
-- 
2.51.2

